CHANGELOG.md

```diff
@@ -3,71 +3,129 @@
+
 
 ### Runtime
 
-* Disable TestExecWindowsOpenHandles on RS5 temporarily. [moby/moby#37679](https://github.com/moby/moby/pull/37679)
-* LCOW: Ensure platform is populated on COPY/ADD. [moby/moby#37563](https://github.com/moby/moby/pull/37563)
-* LCOW: Mount to short container paths to avoid command-line length limit. [moby/moby#37659](https://github.com/moby/moby/pull/37659)
-* LCOW: lazycontext: Use correct lstat, fix archive check. [moby/moby#37356](https://github.com/moby/moby/pull/37356)
-* Lcow: fix debug in startServiceVMIfNotRunning(). [moby/moby#37446](https://github.com/moby/moby/pull/37446)
-* Revendor Microsoft/opengcs @ v0.3.8. [moby/moby#37515](https://github.com/moby/moby/pull/37515)
-* Select polling based watcher for docker log file watcher on Windows. [moby/moby#37412](https://github.com/moby/moby/pull/37412)
-* Updated the go-winio library to release 0.4.8 that has the fix for Windows Container. [docker/cli#1157](https://github.com/docker/cli/pull/1157)
-+ Add --chown flag support for ADD/COPY commands for Windows. [moby/moby#35521](https://github.com/moby/moby/pull/35521)
-+ Adds LinuxMetadata support by default on Windows. [moby/moby#37514](https://github.com/moby/moby/pull/37514)
++ Configure containerd log-level to be the same as dockerd [moby/moby#37419](https://github.com/moby/moby/pull/37419)
++ Add configuration option for cri-containerd [moby/moby#37519](https://github.com/moby/moby/pull/37519)
++ Update containerd client to v1.2.0-rc.1 [moby/moby#37664](https://github.com/moby/moby/pull/37664), [docker/engine#75](https://github.com/docker/engine/pull/75) / [moby/moby#37710](https://github.com/moby/moby/pull/37710)
+
+
+### Security
+
+- Remove support for TLS < 1.2 [moby/moby#37660](https://github.com/moby/moby/pull/37660)
+- Seccomp: Whitelist syscalls linked to `CAP_SYS_NICE` in default seccomp profile [moby/moby#37242](https://github.com/moby/moby/pull/37242)
+- Seccomp: move the syslog syscall to be gated by `CAP_SYS_ADMIN` or `CAP_SYSLOG` [docker/engine#64](https://github.com/docker/engine/pull/64) / [moby/moby#37929](https://github.com/moby/moby/pull/37929)
+- SELinux: Fix relabeling of local volumes specified via Mounts API on selinux-enabled systems [moby/moby#37739](https://github.com/moby/moby/pull/37739)
+- Add warning if REST API is accessible through an insecure connection [moby/moby#37684](https://github.com/moby/moby/pull/37684)
+- Mask proxy credentials from URL when displayed in system info [docker/engine#72](https://github.com/docker/engine/pull/72) / [moby/moby#37934](https://github.com/moby/moby/pull/37934)
+
+
+### Storage drivers
+
+- Fix mount propagation for btrfs [docker/engine#86](https://github.com/docker/engine/pull/86) / [moby/moby#38026](https://github.com/moby/moby/pull/38026)
+
 
 ### Swarm Mode
 
++ Add support for global default address pools [moby/moby#37558](https://github.com/moby/moby/pull/37558) [docker/cli#1233](https://github.com/docker/cli/pull/1233)
```

Comment:
@anshulpundir @dperny please check if there's notable SwarmKit changes since Docker 18.06 that we should mention